tests/api/ask_astro/rest/controllers/test_post_request.py

```diff
@@ -0,0 +1,35 @@
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "request_payload,expected_status,expected_response",
+    [
+        ({"prompt": "Tell me about space"}, 200, {"request_uuid": "test-uuid"}),
+        ({"prompt": "What is quantum mechanics?"}, 200, {"request_uuid": "test-uuid"}),
+        ({}, 400, {"error": "prompt is required"}),
+    ],
+)
+async def test_on_post_request(app, request_payload, expected_status, expected_response):
+    """Test the POST request endpoint behavior based on different input payloads."""
+    with patch("ask_astro.services.questions.answer_question") as mock_answer_question, patch(
+        "ask_astro.clients.firestore.firestore.AsyncClient"
+    ) as mock_firestore:
+        with patch("google.cloud.firestore_v1.Client", new=AsyncMock()):
```

Comment:
Wondering is there any reason we need to add an additional indent on this one?

Reply:
https://github.com/astronomer/ask-astro/pull/77/commits/455e1a42e21005be7bb3d0cf066cf2ff924f97b3